src/templates/layouts/event/index.tsx

```diff
@@ -228,33 +230,26 @@ export const Event = ({
               </p>
             ) : (
               <>
-                {link && (
-                  <p className="vads-u-margin--0">
-                    <a className="vads-c-action-link--green" href={link?.uri}>
-                      {eventCTA && eventCTA != 'more_details'
```

Comment:
Replaced the nested ternary with the `formatEventCTA` utility function which does the same thing.